pkg/mutate/manage_nodeaffinity.go

```diff
@@ -0,0 +1,115 @@
+package mutate
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+
+	offv1alpha1 "github.com/liqotech/liqo/apis/offloading/v1alpha1"
+	liqoconst "github.com/liqotech/liqo/pkg/consts"
+)
+
+// todo: if the clusterSelcetor is empty at NamespaceOffloading creation time, in the NamespaceOffloading Controller
+//       we have to set with a default matchexpression for virtual nodes only, so this is never empty
+
+// This function is global in order to access it with test in test/unit/webhookLogic
+
+// ManageAffinity sets right Pod Affinity according to the PodOffloadingStrategy specified in NamespaceOffloading resource.
+func ManageAffinity(namespaceOffloading *offv1alpha1.NamespaceOffloading, pod *corev1.Pod) {
+	// This newNodeSelector will substitute the old nodeSelector if the strategy is not "LocalPodOffloadingStrategy".
+	newNodeSelector := &corev1.NodeSelector{NodeSelectorTerms: []corev1.NodeSelectorTerm{}}
```

Comment:
I think this may be acceptable in the first place but then I would tend to merge the existing selector with the Liqo one. @giorio94 what do you think about it?

Reply:
yes it is already merged, but I'll make it clearer in the next commit.